react/tic-tac-toe/src/app/screens/LogIn/components/RegisterForm/index.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+import { Field, reduxForm } from 'redux-form';
+import { customInput } from '../Field';
+import { required, minLength, matchesPassword, asyncValidate } from '../../validation';
+import './styles.scss';
+import { normalize } from 'path';
+
+class RegisterForm extends Component {
```

Comment:
This class has no behaviour, I'd define it as a stateless component by using an arrow function

Reply:
:white_check_mark: